packages/common/src/utils/validateFileName.ts

```diff
@@ -29,6 +29,7 @@ export function isValidFileName(fileName: string) {
   return (
     !INVALID_FILENAME_WHITESPACE_REGEX.test(fileName) &&
```

Comment:
we can restrict the fileName length in the regex in itself. somewhat like `{1,64}`